src/migrator/utils.ts

```diff
@@ -152,3 +275,95 @@ export const migrationTablesExist = async (tx: Tx) => {
 	const result = await tx.tableList(where);
 	return result.rows.length === tables.length;
 };
+
+export const initMigrationStatus = async (
+	tx: Tx,
+	migrationStatus: MigrationStatus,
+): Promise<Result | undefined> => {
+	try {
+		return await tx.executeSql(
+			binds`
+INSERT INTO "migration status" ("migration key", "start time", "is backing off")
+SELECT ${1}, ${2}, ${3}
+WHERE NOT EXISTS (SELECT 1 FROM "migration status" WHERE "migration key" = ${4})
+`,
+			[
+				migrationStatus['migration_key'],
+				migrationStatus['start_time'],
+				migrationStatus['is_backing_off'] ? 1 : 0,
+				migrationStatus['migration_key'],
```

Comment:
1. binds`` does only allow a sequence of parameters so that $1, $2, $3, $1 is failling.
2. When I change binds to accept this ordered list of bind keys the tx.executeSql will throw: `DatabaseError: inconsistent types deduced for parameter $1`
This comes from different types deduced from the paramter for the SELECT and the WHERE clause.
@thgreasi so I'm going to just use a duplicate 4th parameter.

Reply:
ok cool :+1: